core/domain/stats_jobs_continuous.py

```diff
@@ -697,8 +757,10 @@ def reduce(key, stringified_values):
 
 class InteractionAnswerSummariesRealtimeModel(
         jobs.BaseRealtimeDatastoreClassForContinuousComputations):
-    # TODO(bhenning): Implement a real-time model for
-    # InteractionAnswerSummariesAggregator.
+    """
+     TODO(bhenning): Implement a real-time model for
+     InteractionAnswerSummariesAggregator.
+    """
```

Comment:
This is not a docstring. It's better to keep it an inline comment using (#) as it was previously.